CloudM.Exercise.WebApi/Controllers/PackageController.cs

```diff
@@ -27,12 +31,63 @@ public async Task<PackageDependencies> Get(string name, string version)
                 PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
             });
 
+            var dependencies = npmPackage.Versions[version].Dependencies;
+
+            foreach (var dep in dependencies)
+            {
+                var subDep = GetDependencies(dep.Key, dep.Value).Result;
+                dependencyTree.Add(dep.Key, new
+                {
+                    Version = dep.Value,
+                    Dependecies = subDep
+                });
+            }
+
             return new PackageDependencies
```

Comment:
Would it not be better if we use status codes according to the result, as opposed to just returning the object itself?